### PR TITLE
Sprint 3 Development

### PR DESCRIPTION
**Pull request by belalakbar**
All the following issues has been resolved.
policycompass/policycompass#153
policycompass/policycompass#152
policycompass/policycompass#151
policycompass/policycompass#150
policycompass/policycompass#149
policycompass/policycompass#145
policycompass/policycompass#141
policycompass/policycompass#130

### CODE REVIEW

**Comment by belalakbar** [comment]
@FabiApfelkern now check.

File: app/modules/fcm/partials/list.html [hunk 1]
```diff
@@ -14,17 +14,17 @@ <h3 class="panel-title">{{ model.title | limitTo: 25}}</h3>
           <div class="panel-body">
             {{ model.dateAddedtoPC | date:'longDate' }}
             <br /><br />
-            <a href="#!/models/{{ model.id }}" type="button" class="btn btn-primary">Details</a>
+            <a href="#/models/{{ model.id }}" type="button" class="btn btn-primary">Details</a>
```
Comment (FabiApfelkern) [inline app/modules/fcm/partials/list.html]: You need to use hashbang mode: #!

**Comment by FabiApfelkern** [comment]
I guess something was merged wrongly. Previous changes were revoked, e.g. the new Hashbang mode, Please merge the current master into your branch. So make a pull origin master BEFORE merging.

**Comment by belalakbar** [comment]
Sorry I don't get your point.

**Comment by belalakbar** [comment]
I did the follow command git pull origin master, now again I commit this or it is fine

**Comment by FabiApfelkern** [comment]
I guess you had merge conflicts for creating this PR. When you resolved them you have chosen the wrong changes. For example: https://github.com/policycompass/policycompass-frontend/commits/Sprint3Development/app/modules/fcm/partials/list.html
Look at this history. I have changed the routing mode in the second last commit and you revoked those changes. This can not happen by accident.

**Comment by belalakbar** [comment]
I got your point. Now fixed it and updated.

**Comment by FabiApfelkern** [comment]
I think there are still major issues here. I tried this PR locally and I am convinced that you have revoked some of your own changes. See the attached images. The screen is further developed in the current master. (https://frontend-stage.policycompass.eu/app/#!/models/create). You should checkout the branch locally and test everything again.

screenshot from 2015-08-13 12-08-05

**Comment by belalakbar** [comment]
I think these issues are due some changes on the files made by you are not consist with my files uploaded. I am working on these and upload again

**Comment by FabiApfelkern** [comment]
You are doing something wrong. It is not about changes I or anyone else did. Can you describe your git workflow?

**Pull request closed without merging**